luminex/webapp/luminex/GuideSetWindow.js

```diff
@@ -338,19 +348,15 @@ function createGuideSetWindow(protocolId, currentGuideSetId, allowEdit) {
 }
 
 function checkGuideSetWindowDirty(name) {
```

Comment:
this parameter looks unused?

Reply:
Yep. I'll get rid of it in a follow-on FB.